FRCore/FRCore/Keychain/SecuredKey.swift

```diff
@@ -22,7 +22,7 @@ public struct SecuredKey {
     /// Public Key of SecuredKey
     fileprivate var publicKey: SecKey
     /// Algorithm to be used for encryption/decryption using SecuredKey
-    fileprivate let algorithm: SecKeyAlgorithm = .eciesEncryptionCofactorX963SHA256AESGCM
+    fileprivate let oldAlgorithm: SecKeyAlgorithm = .eciesEncryptionCofactorX963SHA256AESGCM//.eciesEncryptionCofactorVariableIVX963SHA256AESGCM
```

Comment:
//.eciesEncryptionCofactorVariableIVX963SHA256AESGCM - can we remove the commented code ?